recipes/atomicrex/disabletests.patch

```diff
@@ -0,0 +1,18 @@
+disable tests as they require glibc 2.17
```

Comment:
Instead of disabling tests try doing `export LDFLAGS="$LDFLAGS -lrt"`

Reply:
Unfortunately that still gives the same error: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=78216&view=logs&jobId=10bda42a-d420-5c28-a43f-bc4e66405e5c

Reply:
Can you run `make VERBOSE=1` for more verbosity?

Reply:
Then I get: 
```
$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++  -fvisibility-inlines-hidden -std=c++17 -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I$PREFIX/include -fdebug-prefix-map=$SRC_DIR=/usr/local/src/conda/atomicrex-1.0 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix -O3 -DNDEBUG  -Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,-rpath,$PREFIX/lib -Wl,-rpath-link,$PREFIX/lib -L$PREFIX/lib -lrt -rdynamic CMakeFiles/test_minimizer.cpp.dir/tests/minimizers/test_minimizer.cpp.o  -o test_minimizer.cpp src/arexlib/libatomicrexlib.a $PREFIX/lib/libboost_filesystem.a -lxml2 $PREFIX/lib/libmuparser.so -lnlopt -fopenmp -lgfortran -lgomp -lquadmath -lpthread 
/home/janssen/miniconda3/conda-bld/atomicrex_1570396329992/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.3.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: CMakeFiles/test_minimizer.cpp.dir/tests/minimizers/test_minimizer.cpp.o: undefined reference to symbol 'clock_gettime@@GLIBC_2.2.5'
/home/janssen/miniconda3/conda-bld/atomicrex_1570396329992/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.3.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: /home/janssen/miniconda3/conda-bld/atomicrex_1570396329992/_build_env/bin/../x86_64-conda_cos6-linux-gnu/sysroot/usr/lib/../lib/librt.so: error adding symbols: DSO missing from command line
```

So to me it seems disabling the tests is the easiest way for now.

Reply:
Then you can add, `"-lrt"` to https://gitlab.com/atomicrex/atomicrex/blob/master/cmake/Testing.cmake#L61 by patching.